envs/arrow-env.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+￼  - feedstock : arrow-cpp
```

Comment:
There is some garbage character over here which is also causing github checks to fail.